BluetoothExplorer/Controller/GATT Characteristic Editors/FirmwareRevisionStringCharacteristicViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  FirmwareRevisionStringCharacteristicsViewController.swift
+//  BluetoothExplorer
+//
+//  Created by Carlos Duclos on 6/25/18.
+//  Copyright © 2018 PureSwift. All rights reserved.
+//
+
+import UIKit
+import Bluetooth
+
+final class FirmwareRevisionStringCharacteristicViewController: UIViewController {
+    
+    // MARK: - IB Outlets
+    
+    @IBOutlet private(set) var firmwareTextField: UITextField!
+    
+    // MARK: - Properties
+    
+    var value: GATTFirmwareRevisionString = ""
+    
+    var valueDidChange: ((GATTFirmwareRevisionString) -> ())?
```

Comment:
We need to disable editing if the characteristic only has `read` properties. Check the Battery Level VC.